### ISSUE TITLE
Pause in time task

### ISSUE THREAD

**Issue by micker** [projectfork/Projectfork#1752]
Hello it will be great to add pause system for task time
it possible ?
regards

**Comment by angek** [comment]
Not sure I fully understand but a task is defined by its start and expected completion date. it is not possible to pause tasks in Projectfork.

**Comment by micker** [comment]
When we use time in task it will be good to pause time if we switch to an other task or go to lunch... Now we need to stop and re open it...

On Sat, May 14, 2016 at 4:30 AM, angek < notifications@github.com [notifications@github.com] > wrote:
Not sure I fully understand but a task is defined by its start and expected completion date. it is not possible to pause tasks in Projectfork.

**Comment by angek** [comment]
Understood ;)
Marking it as a feature request

**Comment by micker** [comment]
Very gooooood !
Maybe we can enchance CSS of this popup ??
@angek did you have some news about your gantchart module ?

**Comment by angek** [comment]
Yes i do have some news. Ive updated it to work with Joomla 3.5 and ran some basic initial tests which have been good.
Stay tuned for a further announcement in the next couple of days :)

**Comment by micker** [comment]
cooollll
if you need testeur and french translation ask me
